packages/@aws-cdk/assertions/README.md

```diff
@@ -175,18 +181,20 @@ level, the list of keys in the target is a subset of the provided pattern.
 // }
 
 // The following will NOT throw an assertion error
-assert.hasResourceProperties('Foo::Bar', {
+const expected = {
   Fred: Match.objectLike({
     Wobble: 'Flob',
   }),
-});
+};
+template.hasResourceProperties('Foo::Bar', expected);
 
 // The following will throw an assertion error
-assert.hasResourceProperties('Foo::Bar', {
+const unexpected = {
   Fred: Match.objectLike({
     Brew: 'Coffee',
```

Comment:
Why does this evaluate to `Brew="Coffee"` in the Python translation? I feel like it should be a `:`.

Reply:
Yeah, don't do that. It's better to fix it upstream.